client/tests/integration/sorting.rs

```diff
@@ -52,7 +55,7 @@ fn correct_pagination_assets_after_creating_new_one() {
     let res = test_client
         .request_with_pagination_and_sorting(
             client::asset::by_account_id(account_id.clone()),
-            Pagination::new(Some(1), Some(5)),
+            Pagination::new(None, Some(5)),
```

Comment:
We skipped the `rose` here in the previous implementation. Now, the `rose` is placed at the tail.